.github/workflows/create-release.yaml

```diff
@@ -50,56 +48,54 @@ jobs:
 #          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 #        run: ./scripts/label_validator.sh "RELEASE"
 
-# TODO: uncomment - sec. scanning
-#  bump-sec-scanners-config:
-#    name: Bump sec-scanners-config
-#    needs: validate-release
-#    runs-on: ubuntu-latest
-#
-#    steps:
-#      - name: Checkout code
-#        if: ${{ inputs.sec-scanners-config }}
-#        uses: actions/checkout@v3
-#        with:
-#          fetch-depth: 0
-#
-#      - name: Update sec-scanners-config.yaml
-#        if: ${{ inputs.sec-scanners-config }}
-#        run: scripts/create_scan_config.sh ${{env.IMAGE_REPO}}:${{ inputs.name }} "sec-scanners-config.yaml" ${{ inputs.name }}
-#
-#      - name: Create PR if anything changed
-#        if: ${{ inputs.sec-scanners-config }}
-#        env:
-#          GH_TOKEN: ${{ secrets.BOT_TOKEN }}
-#        run: |
-#          prs=$(gh pr list -A ${{ env.GIT_NAME }} --state open --json headRefName | jq -r '.[] | .headRefName')
-#          if echo $prs | tr " " '\n' | grep -F -q -x ${{ env.BRANCH_NAME }}; then
-#            echo "PR already exists, no need to create a new one"
-#            echo "PR_NUMBER=$(gh pr list --search "base:main head:${{ env.BRANCH_NAME }}" --json number | jq -r '.[] | .number')" >> $GITHUB_ENV
-#          elif [ -z "$(git status --porcelain)" ]; then
-#            echo "Nothing changed, no need to create PR"
-#            echo "PR_NUMBER=-1" >> $GITHUB_ENV
-#          else
-#            PR_STATUS=$(scripts/create_sec_scanner_bump_pr.sh ${{ inputs.name }})
-#            echo "PR_NUMBER=$(echo "$PR_STATUS" | tail -n 1)" >> $GITHUB_ENV
-#          fi
-#
-#      - name: Await PR merge
-#        if: ${{ inputs.sec-scanners-config }}
-#        timeout-minutes: 10
-#        env:
-#          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-#        run: |
-#          if [ "$PR_NUMBER" -gt 0 ]; then
-#            scripts/await_pr_merge.sh
-#            sleep 15
-#          else
-#            echo "Step skipped"
-#          fi
+  bump-sec-scanners-config:
+    name: Bump sec-scanners-config
+    needs: validate-release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        if: ${{ inputs.sec-scanners-config }}
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Update sec-scanners-config.yaml
+        if: ${{ inputs.sec-scanners-config }}
+        run: scripts/create_scan_config.sh "sec-scanners-config.yaml" ${{ inputs.name }}
+
+      - name: Create PR if anything changed
```

Comment:
There needs to be a step to save the latest commit ref created by this step. The ref is needed in other jobs, e.g. `create draft release`, so that the most recent commit is not omitted, since github actions only sees the most recent commit from the time the workflow was run. Example how to do it https://github.com/kyma-project/btp-manager/blob/dc810239ffd4dc92c52ba38235f23194a8b41f0c/.github/workflows/create-release.yaml#L108